frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -84,19 +86,32 @@ type FacetAccordionItemProps = AccordionItemProps & {
    refinedCount: number;
    productList: ProductList;
    isExpanded: boolean;
+   length: number;
 };
 
 export const FacetAccordionItem = forwardRef<FacetAccordionItemProps, 'div'>(
-   ({ attribute, refinedCount, productList, isExpanded, ...props }, ref) => {
+   (
+      { attribute, refinedCount, productList, isExpanded, length, ...props },
+      ref
+   ) => {
       const { items } = useFilteredRefinementList({ attribute });
       const { hits } = useHits();
       const isProductListEmpty = hits.length === 0;
       const hasApplicableRefinements = items.length > 0;
-      const isDisabled = isProductListEmpty || !hasApplicableRefinements;
+      let isDisabled = isProductListEmpty || !hasApplicableRefinements;
+      let isHidden = !hasApplicableRefinements && !isProductListEmpty;
 
-      const formattedFacetName = formatFacetName(attribute);
-      const isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      if (isHidden && attribute !== 'worksin') {
+         checked += 1;
+      }
+
+      if (checked === length) {
+         isDisabled = true;
+         isHidden = false;
+         checked = 0;
+      }
```

Comment:
So will this cause the last facet to be rendered and disabled if there are no applicable facets?

Reply:
Yes

Reply:
That is what I intended because we were just looking to display some disabled facet but based on the bugs...something isn't right.

Reply:
That is not what is happening...I'll look into it

Reply:
This is a possible fix 2f96e03249b012ad3a9018e9e50d1e9161eb0252